database/drivers/driver.go

```diff
@@ -0,0 +1,20 @@
+package drivers
+
+import (
+	"log"
+
+	"gnorm.org/gnorm/database"
+)
+
+// Driver defines the base interface for databases that are supported by gnorm
+type Driver interface {
+	Name() string
```

Comment:
name isn't used anywhere right now, and I think that's correct.  I like that the names for the config file are defined by the cli package, not by the drivers themselves.  keeping this field will confuse people into thinking it defines the configuration value, I think.  Let's drop it.